Client/src/components/common/LabeledInput/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Input from '../Input';
+import Label from '../Label';
+
+const LabeledInput = props => {
+  return (
+    <Label input={Input} {...props} className={props.container} />
+  )
+};
+
+export default LabeledInput;
+
```

Comment:
Look at this idea: 
```
function labeled(props) {
  const { id, labelText, onChange, placeholder } = { ...props };
  return (
      <Label htmlFor={id} labelText={labelText} input = {<Input id={id} placeholder={placeholder} onChange={onChange} />}/>
  );
}

```

Reply:
it's better to separate each component in a file and then call it 
so you can use it somewhere else